### ISSUE TITLE
Avoid unloading MSL in BESTEST CI tests

### ISSUE THREAD

**Issue by mwetter** [ibpsa/modelica-ibpsa#1554]
This issue is to apply
```diff
diff --git a/IBPSA/Resources/Data/BoundaryConditions/Validation/BESTEST/generateResults.py b/IBPSA/Resources/Data/BoundaryConditions/Validation/BESTEST/generateResults.py
index b6c4f28f9..eb825caaf 100755
--- a/IBPSA/Resources/Data/BoundaryConditions/Validation/BESTEST/generateResults.py
+++ b/IBPSA/Resources/Data/BoundaryConditions/Validation/BESTEST/generateResults.py
@@ -249,8 +249,6 @@ def _simulate(spec):
     s = Simulator(spec["model"])
     # Add all necessary parameters from Case Dict
     s.addPreProcessingStatement("OutputCPUtime:= true;")
-    s.addPreProcessingStatement("// For Dymola 2022 (or higher) unload MSL so that MSL from uses statement is loaded")
-    s.addPreProcessingStatement("if DymolaVersionNumber() <> 2021.0 then eraseClasses({\"Modelica\"}); end if;")
     s.setSolver(spec["solver"])
     if 'parameters' in spec:
         s.addParameters(spec['parameters'])

```
This is needed for BuildingsPy above commit 9085d1da75123ac028993d31697acf58a002a885.
I will apply this directly to the master once the above commit of BuildingsPy is merged to the BuildingsPy master.

This change will cause older branches to fail, until this change is merged into them.
